x-pack/plugins/infra/public/types.ts

```diff
@@ -75,7 +74,6 @@ export interface InfraClientStartDeps {
   embeddable?: EmbeddableStart;
   osquery?: unknown; // OsqueryPluginStart;
   share: SharePluginStart;
-  lens: LensPublicStart;
```

Comment:
we might want to leave this for now as we are thinking to use Lens Embeddables for other things